build.gradle

```diff
@@ -239,6 +239,7 @@ task distCompiler(type: Copy) {
     dependsOn ':utilities:jar'
     dependsOn ':klib:jar'
     dependsOn ':sharedJar'
+    dependsOn ':defaultLibraries:build'
```

Comment:
The `build` task will run tests. May be create a `defaultLibraries:assemble` task depending on `assemble` tasks of subprojects and depend on it?